src/ripple/app/tx/impl/PayChan.cpp

```diff
@@ -252,6 +263,16 @@ PayChanCreate::doApply()
         (*slep)[sfOwnerNode] = *page;
     }
 
+    // Add PayChan to the recipient's owner directory
+    if (ctx_.view().rules().enabled(fixPayChanRecipientOwnerDir))
+    {
+        auto page = dirAdd(ctx_.view(), keylet::ownerDir(dst), slep->key(),
```

Comment:
`page` could be `const` (and similarly on line 259).

Reply:
Fixed in 613db91af `Add const in some places & fix typo`